packages/core/src/lib/Select/SelectBeta/Select.test.tsx

```diff
@@ -0,0 +1,189 @@
+
+    beforeEach(() => {
+        container = document.createElement('div');
+
+        render(<BetaSelectDisabled onChange={testClickEvent} />, {
+            container: document.body.appendChild(container),
+        });
+    });
+    afterEach(() => {
+        cleanup();
+    });
+
+    it('renders the component', () => {
+        const element = container.querySelector(
+            `[data-testid="${testWrapperId}"]`,
+        );
+        expect(element).not.toBeNull();
+    });
+
+    it('renders label correctly', () => {
+        const element = container.querySelector(
+            `[data-testid="${testLabelId}"]`,
+        );
+        expect(element).not.toBeNull();
+        expect(element?.textContent).toBe(labelText);
+    });
+
+    it('renders no dropdown list', () => {
+        const element = container.querySelector(
+            `[data-testid="${testSelectButton}"]`,
+        );
+        testSelectClick(true, testSelectButton, element as HTMLButtonElement);
+    });
+
+    it('renders search box', () => {
+        const element = container.querySelector(
+            `[data-testid="${testSelectButton}"]`,
+        );
+        testSelectClick(true, testSelectButton, element as HTMLButtonElement);
+        const searchInput = container.querySelector(
+            `[data-testid="${testSearchInput}"]`,
+        );
+        expect(searchInput).toBeNull();
+    });
+});
+
+describe('Renders - Select Beta No Search', () => {
+    const labelText = BetaSelectNoSearch?.args?.label;
+
+    beforeEach(() => {
+        container = document.createElement('div');
+
+        render(<BetaSelectNoSearch onChange={testClickEvent} />, {
+            container: document.body.appendChild(container),
+        });
+    });
+
+    afterEach(() => {
+        cleanup();
+    });
+
+    it('renders the component', () => {
+        const element = container.querySelector(
+            `[data-testid="${testWrapperId}"]`,
+        );
+        expect(element).not.toBeNull();
+    });
+
+    it('renders label correctly', () => {
+        const element = container.querySelector(
+            `[data-testid="${testLabelId}"]`,
+        );
+        expect(element).not.toBeNull();
+        expect(element?.textContent).toBe(labelText);
+    });
+
+    it('renders dropdown list', () => {
+        const element = container.querySelector(
+            `[data-testid="${testSelectButton}"]`,
+        );
+        testSelectClick(false, testSelectButton, element as HTMLButtonElement);
+    });
+
+    it('renders search box', () => {
+        const element = container.querySelector(
+            `[data-testid="${testSelectButton}"]`,
+        );
+        testSelectClick(false, testSelectButton, element as HTMLButtonElement);
+        const searchInput = container.querySelector(
+            `[data-testid="${testSearchInput}"]`,
+        );
+        // search is hidden test
+        expect(searchInput).toHaveAttribute('aria-hidden', 'true');
+    });
+});
```

Comment:
nice tests, how did the coverage turn out?

Reply:
will check 🙌
will add a bit more tests 😋

Reply:
![image](https://user-images.githubusercontent.com/35369843/185473006-95c2ccac-b621-497e-a695-a4c94f0bc7f3.png)
Not bad, Needs more tests so that future modifications won't break this